components/TextFields/src/MDCTextInputControllerFullWidth.m

```diff
@@ -424,6 +424,22 @@ - (void)setCharacterCountViewMode:(UITextFieldViewMode)characterCountViewMode {
   }
 }
 
+- (void)setDisabledColor:(UIColor *)disabledColor {
```

Comment:
My understanding is you need __unused for suppressing 'unused variable' warnings. This doesn't trigger one in my Xcode 9. Are you seeing it? It's defined in a protocol so it has to be implemented. This particular class doesn't _yet_ need it.